control-plane/msp-operator/src/main.rs

```diff
@@ -435,93 +435,101 @@ impl ResourceContext {
             // we updated the resource as an error stop reconciliation
             return Err(Error::ReconcileError { name: self.name() });
         }
-        match self
+        let check_if_block_device = if self.spec.disks[0].starts_with("aio:///") {
+            let path = normalize_disk(&self.spec.disks[0][6..]);
+            !std::path::Path::new(&path).exists()
```

Comment:
It looks like you are checking if the file exists within the context of the MSP operator. Dont want to check that it's available on the target node?

Reply:
Ah, yes, you are in fact correct.